src/test/scala/com/danielasfregola/twitter4s/entities/DeserializationRoundtripSpec.scala

```diff
@@ -0,0 +1,81 @@
+package com.danielasfregola.twitter4s.entities
+
+import com.danielasfregola.twitter4s.helpers.{FixturesSupport, JsonDiffSupport}
+import org.json4s.native.Serialization.writePretty
+import org.json4s.native.{JsonParser, Serialization}
+import org.json4s.{JNothing, JValue}
+import org.specs2.matcher.{Expectable, Matcher}
+import org.specs2.mutable.Specification
+import org.specs2.specification.core.Fragment
+
+import scala.reflect._
+
+class DeserializationRoundtripSpec extends Specification with FixturesSupport with JsonDiffSupport {
+
+   "JSON deserialization" should {
+
+     def roundtripTest[T <: AnyRef : Manifest](jsonFile: String): Fragment = {
+
+       val className = classTag[T].runtimeClass.getSimpleName
+
+       s"round-trip successfully for $className" in {
+         val originalJson = load(jsonFile)
+
+         val deserializedEntity = Serialization.read[T](originalJson)
+
+         val serializedJson = Serialization.writePretty[T](deserializedEntity)
+
+         originalJson must beASubsetOfJson(serializedJson)
+       }
+     }
+
+     roundtripTest[User]("/twitter/rest/users/user.json")
```

Comment:
[Future improvements not related to this PR] maybe we can use more json files to enrich we analyze enough json shapes